packages/hurl/src/http/response_ip.rs

```diff
@@ -0,0 +1,31 @@
+/*
+ * Hurl (https://hurl.dev)
+ * Copyright (C) 2025 Orange
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+use std::str::FromStr;
+
+#[derive(Clone, Debug, Default, Eq, PartialEq)]
+pub struct IpAddr {
+    pub(crate) raw: String,
+}
+
+impl IpAddr {
+    #[allow(dead_code)]
+    fn to_ip_addr(&self) -> Result<std::net::IpAddr, String> {
```

Comment:
If possible, we can reuse [AddrParseError](https://doc.rust-lang.org/beta/core/net/struct.AddrParseError.html) as the error

```rust
fn to_ip_addr(&self) -> Result<std::net::IpAddr, AddrParseError>
```